src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -353,6 +358,11 @@ public static bool TryParse(string? version, out LanguageVersion result)
                     result = LanguageVersion.CSharp8;
                     return true;
 
+                case "9":
+                case "9.0":
```

Comment:
Some tests should be updated for this.
Look at ` LanguageVersionAdded_Canary` and subsequent tests, which should fail when new version is added.